src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue

```diff
@@ -65,34 +69,40 @@
     { label: 'does not exist', value: 'does_not_exist' }
   ])
 
+  const showLabelApplicationAccelerator = computed(() => {
+    if (props.showRequiresApplicationAcceleratorInDescription) return ''
```

Comment:
Podemos alterar para
```suggestion
    if (props.hasApplicationAcceleratorActive) return ''
```
?

O texto me parece contra-intuitivo. Se `showRequiresApplicationAcceleratorInDescription` for true, parece que deveríamos mostrar o texto ... mas é exatamente o contrário do que estamos fazendo aqui.